test/nuts/local/replacements/replacements.nut.ts

```diff
@@ -42,16 +26,7 @@ describe('e2e replacements test', () => {
     // Hack: rewrite the file replacement locations relative to the project
     const projectJsonPath = path.join(session.project.dir, 'sfdx-project.json');
     const original = await fs.promises.readFile(projectJsonPath, 'utf8');
-    await fs.promises.writeFile(
-      projectJsonPath,
-      original
-        // we're putting this in a json file which doesnt like windows backslashes.  The file will require posix paths
```

Comment:
the NUT now uses relative filenames for these.  They work using absolute, but I expect that to be less common.